CMakeLists.txt

```diff
@@ -89,25 +89,18 @@ if(NOT MSVC)
 endif()
 
 if(Boost_FOUND)
-  get_property(SOCI_COMPILE_DEFINITIONS
-    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    PROPERTY COMPILE_DEFINITIONS)
-
-  list(APPEND SOCI_COMPILE_DEFINITIONS "HAVE_BOOST=1")
-  list(APPEND SOCI_COMPILE_DEFINITIONS "BOOST_ALL_NO_LIB")
```

Comment:
I just checked the meaning of `BOOST_ALL_NO_LIB`, I was thinking it was soci specific. So I keep it